### ISSUE TITLE
Is there anyway limit fps without forceSetTimeOut=True

### ISSUE THREAD

**Issue by wudizhangzhi** [phaserjs/phaser#6222]
if i wanna limit fps to 30. i can only set forceSetTimeOut=True。But it's not good as using requestAnimationFrame and count elapsed.

Is there anyway in phaser to perform  like the code below?

````javascript
let fps = 30
let fpsInterval = 1000 / fps
let fpsLast

let transport = undefined

function setFps() {
    fpsLast = new Date().getTime()
}

function setScreen() {
    canvas.width = ctxWidth;
    canvas.height = ctxHeight;
}

function render() {
    frameId = window.requestAnimationFrame(render)
    let now = new Date().getTime()
    let elapsed = now - fpsLast;
    if (elapsed > fpsInterval) {
        fpsLast = now - (elapsed % fpsInterval);
        if(gameRunning){
            main()
        }
    }
}
````

**Comment by photonstorm** [comment]
This feature is available in 3.60 only, it cannot be done in 3.55.

**Comment by wudizhangzhi** [comment]
> This feature is available in 3.60 only, it cannot be done in 3.55.

thx! 
Look forward to the day of release
